cli/js/lib.deno.ns.d.ts

```diff
@@ -29,6 +29,10 @@ declare namespace Deno {
   /** The current process id of the runtime. */
   export const pid: number;
 
+  /** The pid of the current process's parent.
+   * Ignored on Windows. */
+  export const ppid: number | null;
```

Comment:
APIs must be added to cli/js/lib.deno.unstable.d.ts first. You can follow the example of 86f92e04c79be81f98e5899638cb6fdb29a4fa64

Reply:
pidParent? parentPid?

Reply:
> pidParent? parentPid?

Given that we already have pid, ppid makes sense no? Kinda odd if one is the POSIX name and the other is not 🤔

Reply:
- I've made `Deno.ppid` unstable.
- I'm keeping the property name for now.